LondonBritishLibrary/orient/BLorient7946.xml

```diff
@@ -128,6 +132,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <decoNote xml:id="b2" type="Cover"><material key="leather">Half leather.</material></decoNote>
                             </binding>
                         </bindingDesc>
+                        <accMat xml:id="accMat1">
+                            A loose piece of paper has been attached after <locus target="#127"/> containing the number of entries given 
+                            corresponding to each letter. This information is repeated here as given in <locus target="#2v"/> (cf. <ref target="#ms_i1"/>)
```

Comment:
here "given" or "corresponding" is to be deleted?
Maybe better "each Amharic letter" (I was thinking for a while as to what letter is meant).